packages/xo-vmdk-to-vhd/src/ova.js

```diff
@@ -342,7 +342,7 @@ export async function parseOVAFile(
           return parseGzipFromEnd(start, end, fileSlice, header)
         }
       }
-      data.tables[header.fileName] = await readVmdkGrainTable(readFile)
+      data.tables[header.fileName] = readVmdkGrainTable(readFile)
```

Comment:
Where has this `await` moved to?

Who is using this value?

Reply:
it's here: 
https://github.com/vatesfr/xen-orchestra/blob/4264e34ffd437daf331bc5790520fec1c9005190/packages/xo-web/src/common/xo/index.js#L1543